index.js

```diff
@@ -864,6 +849,25 @@ function flameGraph (opts) {
   return chart
 }
 
+function adjustForHighDpiScreen (canvas, w, h) {
+  // Adjust canvas for high DPI screens
+  // - Size the image up N× using attributes
+  // - Squash it down N× using CSS
+  // - Scale the context so 1px in all subsequent draw operations means Npx
+
+  var pixelRatio = window.devicePixelRatio
+
+  if (pixelRatio > 1) {
```

Comment:
I agree,
and since `window.devicePixelRatio` is supported by basically [100% of the browsers around](https://caniuse.com/#feat=devicepixelratio) we can skip the check altogether and always set the canvas attributes.

Reply:
Good call! sorted now